roles/redfish_storage_volume/molecule/RAID6/converge.yml

```diff
@@ -0,0 +1,80 @@
+---
+- name: Ansible - Role- redfish_storage_volume - [Check Mode] [Idempotency] Validate creating RAID6 VD
+  hosts: all
+  vars:
+    failure: {}
```

Comment:
Can you use the vars with a role name prefix

Reply:
Done